MathLib/LinAlg/Eigen/EigenLinearSolver.h

```diff
@@ -0,0 +1,80 @@
+/**
+ * \copyright
+ * Copyright (c) 2012-2015, OpenGeoSys Community (http://www.opengeosys.org)
+ *            Distributed under a Modified BSD License.
+ *              See accompanying file LICENSE.txt or
+ *              http://www.opengeosys.org/project/license
+ *
+ */
+
+#ifndef EIGENLINEARSOLVER_H_
+#define EIGENLINEARSOLVER_H_
+
+#include <vector>
+
+#include <boost/any.hpp>
```

Comment:
I don't see where boost::any is used, and the std::vector too. If unused -> remove.